packages/block-library/src/more/index.js

```diff
@@ -25,7 +25,7 @@ export const name = 'core/more';
 export const settings = {
 	title: _x( 'More', 'block name' ),
 
-	description: __( 'Want to show only an excerpt of this post on your home page? Use this block to define where you want the separation.' ),
+	description: __( 'Do you want to show only an excerpt of this post on your homepage? Then use this block to define where the separation should take place.' ),
```

Comment:
A few thoughts:

1. the leading question only very indirectly explains what this does; we should just outright state what the block does and how it works.
2. should we be saying "post" and "homepage"? Some people don't use the archives listing on their homepage and Gutenberg edits more than just post content.

Here's my suggestion:

```suggestion
	description: __( 'Do you want to show only an excerpt of this post on your homepage? Then use this block to define where the separation should take place.' ),
	description: __( 'Mark the excerpt of this content. Content before this block will be shown in the excerpt on your archives page.' ),
```

Reply:
I guess we could use "signify" instead of "mark" if that's an unclear verb here. 🤷‍♂️